vagrant/tutorial/post-reboot-bootstrap.py

```diff
@@ -102,10 +103,20 @@
 check_call(['truncate', '--size', '1G', '/var/opt/flocker/pool-vdev'])
 check_call(['zpool', 'create', 'flocker', '/var/opt/flocker/pool-vdev'])
 
+
+check_call(['mkdir', '/etc/flocker'])
+check_call(['cp', '/home/vagrant/credentials/cluster.crt',
+            '/etc/flocker/cluster.crt'])
+with file("/etc/flocker/agent.yml", "a") as f:
+    f.write(json.dumps({
+        "control-service": {"hostname": "172.16.255.250"},
+        "dataset": {"backend": "zfs"},
+        "version": 1
+    }))
+
 # Move SSH private key into place so ZFS agent can use it until we remove
```

Comment:
Oh, huh, there's this.

In that case... how about putting *all* the logic here, instead of spread out over Vagrantfile and here? Then you say "if I'm .250, set up control service and agent 1, if I'm .251 set up agent 2". Or alternatively the control service logic should be in the Vagrantfile. But they definitely shouldn't be spread out over two files.